### PR TITLE
Integrate gtest

### PR DESCRIPTION
**Pull request by Tina66**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.11)
+project(handy_rpc)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules")
+
+include(ExternalProject)
+include(GtestExternalProject)
+
+#head file path
```
Comment (jovany-wang) [inline CMakeLists.txt]: header files?

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.11)
+project(handy_rpc)
```
Comment (jovany-wang) [inline CMakeLists.txt]:
```suggestion
project(dmeta)
```

Comment (Tina66) [reply CMakeLists.txt]: thank for your review!

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.11)
+project(dmeta))
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules")
+
+include(ExternalProject)
+include(GtestExternalProject)
+
+#header files path
```
Comment (jovany-wang) [inline CMakeLists.txt]:
```suggestion
# header files path
```

Comment (Tina66) [reply CMakeLists.txt]: This is a txt file, should it be strict like this?

**Pull request merged**